Extension/src/main.ts

```diff
@@ -146,7 +146,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<CppToo
     if (shouldActivateLanguageServer) {
         await LanguageServer.activate();
     } else if (isIntelliSenseEngineDisabled) {
-        LanguageServer.registerCommands(false);
+        LanguageServer.registerCommands(false, /* isRelatedFilesApiEnabled */ false);
```

Comment:
Another idea is that you could have this second parameter be optional and only add the registration if the variable is true. Up to you though.

Reply:
Changed to an optional parameter.